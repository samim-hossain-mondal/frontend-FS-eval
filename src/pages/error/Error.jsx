import React from 'react';
import './Error.css';

export default function Error() {
  return (
    <div className='error-text'>
      <h1>Oops... Something wrong happened</h1>
    </div >
  );
}