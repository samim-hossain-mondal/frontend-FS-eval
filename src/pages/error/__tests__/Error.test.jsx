import React from 'react';
import { render } from '@testing-library/react';
import Error from '../Error';

describe('Error', () => {
  it('should render successfully', () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});