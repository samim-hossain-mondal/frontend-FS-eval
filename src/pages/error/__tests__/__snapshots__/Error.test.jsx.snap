// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render successfully 1`] = `
<div>
  <div
    class="error-text"
  >
    <h1>
      Oops... Something wrong happened
    </h1>
  </div>
</div>
`;
